### ISSUE TITLE
Gut Microbiome Alterations following Postnatal Iron Supplementation Depend on Iron Form and Persist into Adulthood

### ISSUE THREAD

**Issue by Oziomaekeh** [waldronlab/BugSigDBcuration#278]

**Comment by imalovee** [comment]
Hello @Oziomaekeh , there is no link here to access the paper, plus I'd love to check it out. Thanks.

**Comment by SvetlanaUP** [comment]
Gut Microbiome Alterations following Postnatal Iron Supplementation Depend on Iron Form and Persist into Adulthood - McMillen et al - Nutrients
https://www.mdpi.com/2072-6643/14/3/412

@Oziomaekeh since you added this paper, would you like to curate it?

**Comment by Oziomaekeh** [comment]
Yes pls

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Oziomaekeh** [comment]
Yes please

**Comment by SvetlanaUP** [comment]
@Oziomaekeh is this study curated and ready for review? https://github.com/waldronlab/BugSigDBcuration/issues/235

**Comment by SvetlanaUP** [comment]
This one is available for curation.

**Comment by keamybams** [comment]
I would love to curate this. Thanks. Ruth Bamgbose.

**Comment by keamybams** [comment]
Hello @SvetlanaUP, This is ready for review https://bugsigdb.org/Study_999 . Thank you.

**Comment by KateRasheed** [comment]
Good morning @SvetlanaUP . I would like to review this paper please.

**Comment by KateRasheed** [comment]
Good morning @imalovee @keamybams @Buraah. Welldone on this curation. Here are few things to note in this study.

- Please leave the antibiotics exclusion blank
- Please remove Fig. 2d and 3d from experiment 1, 2 signature sources. It's for FC vs FS. You can use the supplementary figures and tables as the source
- Please replace Ruminococcus with Ruminococcus 2 and Ruminiclostridium with Ruminiclostridium 6 in experiment 1 signature 1
- Please replace Massiliimalia with GCA-900066225 (experiment 1, 2, 4, 5, 6). Then GCA−900066575 for uncultured Clostridium sp. in experiment 3, 6 signature 2, experiment 4 signature 1. I don't think they are related (we would confirm this during the next office hour).
- In experiment 2 signature 2, Bacteroides, Dubosiella is missing
- There are some missing taxa not curated from Figure S3 (experiment 3) and Figure S6 (experiment 6)
- For Experiment 6 signature 1, please change Deferribacteres to the phylum

Thank you so much.

**Comment by Buraah** [comment]
Hi @SvetlanaUP, I only briefly went through the study to review and change the data transformation while doing the cleanup. So, I'll take some time to go through it again and then come back to the comments.

**Comment by Buraah** [comment]
Yes, @SvetlanaUP, I've seen the comments. I'm saying that I'm not the initial curator, so the study is new to me, but I will take time to go through it and implement the changes @KateRasheed listed in the comments.

I'll leave a comment when it's ready for a final review.✨

**Comment by Buraah** [comment]
Hi @KateRasheed,
All corrections have been made. Please take a second look.

Thank you.

**Comment by KateRasheed** [comment]
Thank you so much for effecting these corrections @Buraah .

This review is complete @SvetlanaUP

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_999 reviewed.
